src/include/ipc4/base_fw.h

```diff
@@ -399,7 +399,9 @@ enum ipc4_hw_config_params {
 	/* Size of a single memory bank (EBB) in bytes */
 	IPC4_EBB_SIZE_BYTES_HW_CFG        = 9,
 	/* UAOL capabilities */
-	IPC4_UAOL_CAPS_HW_CFG            = 10
+	IPC4_UAOL_CAPS_HW_CFG            = 10,
+	/* Mic privacy capabilities */
+	IPC4_PRIVACY_CAPS_HW_CFG         = 11
```

Comment:
I would re-name these to be more specific to:
IPC4_INTEL_MIC_PRIVACY_CAPS_HW_CFG (and IPC4_INTEL_UAOL_CAPS_HW_CFG).

it looks to me that we are passing the raw content of the DfMICPVCP register, which is pretty much Intel specific

Reply:
Can we rename this to IPC4_INTEL_MIC_PRIVACY_CAPS_HW_CFG. I don't think other vendors will have MICPVCP register in their SoC.